libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -98,17 +107,53 @@ private GutenbergContainerFragment getGutenbergContainerFragment() {
         return mRetainedGutenbergContainerFragment;
     }
 
+    public Bundle getTranslations() {
+        Bundle translations = new Bundle();
+        Locale defaultLocale = new Locale("en");
```

Comment:
This code will always return English translation, is that idea?

Reply:
The english locale yes, the idea is for getTranslations to return a mapping of `english string => [ translated string ]` so we feed it to the @wordpress/i18n lib.